src/main/java/org/icgc/argo/workflow_management/wes/NextflowService.java

```diff
@@ -266,15 +265,18 @@ private CmdKubeRun createCmd(@NonNull Launcher launcher, @NonNull RunParams para
 
     // Dynamic engine properties/config
     val workflowEngineParams = params.getWorkflowEngineParams();
-    val cluster  =  Objects.nonNull(params.getWorkflowParams().get("cluster"))? params.getWorkflowParams().get("cluster").toString(): "cumulus";
+    val cluster =
+        Objects.nonNull(params.getWorkflowParams().get("cluster"))
+            ? params.getWorkflowParams().get("cluster").toString()
+            : "default";
```

Comment:
Thanks for removing the hardcoded cumulus references.

Renaming a cluster to `default` is an interesting compromise - this ensures every request will be scheduled to run, even if the submitted cluster name is incorrect. I think this could be problematic for some users though... if there was an intention to run the workflow in a specific cluster but they typo the name, then suddenly the workflow will be found running in the default cluster. They won't know why either, completely arbitrary decision by the manager without any communication on the request.

For our use case, this could be good enough to unblock our data processing, so I am willing to approve this if we commit to a follow up effort to make this return an error if the cluster is not found. I think it would be correct to put a workflow request into an error state if the requested cluster is not found in the configuration.

Reply:
I misread this check. This is just checking if the request includes the cluster parameter, not checking if we know the given cluster. There is no error or anything that needs changing here. Now that we default to `default` there is no concern.

My only suggestion would be that we need to include some updates to our documentation to explain how to configure clusters. It is now important that there is one cluster named default since the code relies on that specific name.

Reply:
Updated the readme with this info.